src/frontend/src/btc/components/transactions/BtcTransactions.svelte

```diff
@@ -26,7 +27,7 @@
 <TransactionsSkeletons loading={$btcTransactionsNotInitialized}>
 	{#each $sortedBtcTransactions as transaction (transaction.data.id)}
 		<div transition:slide={SLIDE_DURATION}>
-			<BtcTransaction transaction={transaction.data} />
+			<BtcTransaction transaction={transaction.data} token={$tokenWithFallback} />
```

Comment:
Mmm the fallback is Ethereum, is this a good idea or should we assert that the token is given and BTC?

Reply:
you mean `$token ?? BTC_MAINNET_TOKEN` , instead of `$tokenWithFallback`? if I understood correctly, then yes, totally agreed!

I was just trying to respect the old logic, that is flawed too...

Reply:
Great. Yes, either your suggestion or not showing the transaction unless it's bitcoin. One or the other yes.

Reply:
done!